test/tscc/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "optimized-test",
+  "version": "0.0.0",
+  "dependencies": {
+    "@tscc/tscc": "^0.6.4"
```

Comment:
The bad news here: It's using a slightly different version of TypeScript (3.9?) right now.